testsuite/resilient.py

```diff
@@ -54,3 +54,9 @@ def accounts_create(client, params):
             client.accounts.delete(client.accounts.read_by_name(params["name"]).entity_id)
             time.sleep(2)
         raise err
+
+
+@backoff.on_exception(backoff.fibo, ApiClientError, max_tries=8, jitter=None)
```

Comment:
I would like more if the `max_tries` was lower to "Fail faster" in case the 3scale is down, but I am ok with the current implementation too.